### PR TITLE
7571 genre screen

### PR DESCRIPTION
**Pull request by vietdung1996**
## [Pull request title]
7571_genre_screen
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7571
#### Screenshots (nếu có)
image

### CODE REVIEW

**Comment by vietdung1996** [comment]
ready

File: app/src/main/java/com/example/dung/music_59/ui/adapter/TracksAdapter.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        mContext = context;
+        mTracks = tracks;
+        mInflater = LayoutInflater.from(mContext);
+    }
+
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        View view = mInflater.inflate(R.layout.custom_item_genres, viewGroup, false);
+        return new ViewHolder(view,mTrackListener);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/dung/music_59/ui/adapter/TracksAdapter.java]: format code

File: app/src/main/res/layout/activity_track_genres.xml [hunk 2]
```diff
@@ -1,9 +1,58 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_margin="@dimen/dp_5"
+            app:layoutManager="android.support.v7.widget.LinearLayoutManager"/>
+
+    </android.support.v4.widget.NestedScrollView>
+
+</android.support.design.widget.CoordinatorLayout>
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_track_genres.xml]: remove 2 blank line

**Comment by vietdung1996** [comment]
ready

**Pull request merged**